addusers.py

```diff
@@ -171,6 +171,50 @@ def grant_role(self, auth_url, project_id, user_id, role_id):
         return self.keystone.session.put(url)
 
 
+def manual_testing(header):
+    """Testing changes to the code that doesn't involve the form
+    """
+    print "...TESTING..."
+    tests = []
+    tests.append(header)
+    num_tests = int(raw_input("Number of tests: "))
+    for idx in range(num_tests):
+        index_length = len(header)
+        index_flag = raw_input("New Project?    [y/n] ")
+        if index_flag == 'y':
+            index_length -= 4
+        entry = [0] * len(header)
+        for i in range(len(entry)):
+            entry[i] = raw_input(header[i] + "= ")
+        tests.append(entry)
+    return tests
+
+
+def manual_fix(rows, header):
```

Comment:
@ltomatsu feel free to argue against any comments if you disagree, in this case I just don't see a use for this function.  Confirming the user input is a good idea though.